lib/Conversion/TorchToStablehlo/Basic.cpp

```diff
@@ -1833,6 +1833,37 @@ LogicalResult ConvertAtenOp<AtenRemainderTensorOp>::matchAndRewrite(
   return success();
 }
 
+// AtenFmodTensorOp
+// torch.fmod(a, b) == a - a.div(b, rounding_mode="trunc") * b
```

Comment:
As your description, maybe it's a good idea to write a decompisition on torch dialect.

Reply:
For linalg, it decomposes directly to linalg dialect at a similar position. For ONNX, I'm not sure have any impact. I will try to implement it on torch decompisition to see if there are any issues.